doc/source/installation.rst

```diff
@@ -6,7 +6,27 @@ Installation of latest dev version from the source code::
 
     $ git clone https://github.com/Tendrl/bridge_common.git
     $ cd bridge_common
+    $ yum install http://dl.fedoraproject.org/pub/epel/7/x86_64/p/python-pip-7.1.0-1.el7.noarch.rpm
```

Comment:
I dont think we should have distro specific details in install files. 

Either make install files for centos, fedora, ubuntu, or add all those details here